job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1738,6 +1738,22 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /*
```

Comment:
The GroovyDoc comment block needs to start with `/**`. Fix this for all your pull requests.